test/powershell/Modules/Microsoft.PowerShell.Management/Get-ChildItem.Tests.ps1

```diff
@@ -60,6 +60,28 @@ Describe "Get-ChildItem" -Tags "CI" {
             $file.Count | Should be 1
             $file.Name | Should be "pagefile.sys"
         }
+        It "Should continue enumerating a directory when a contained item is deleted" -Skip:($IsWindows) {
```

Comment:
But the underlying behavior is not the same on all platforms. On Unix, if a file is deleted or renamed during enumeration over the result of `DirectoryInfo.EnumerateFiles()`an exception is thrown when attempting to access a property on the `FileSystemInfo` object. On Windows, no exception is thrown, the property is successfully accessed, and the enumeration goes happily on.

The test hook code does not force an error. It sets up the _condition_ which may or may not trigger an error. If the same condition triggers an error on one platform but not on another, how are the tests for _how the error is handled_ expected to behave the same on both platforms?